system/jlib/jptree.cpp

```diff
@@ -1108,6 +1141,15 @@ void CPTValue::deserialize(MemoryBuffer &src)
     }
 }
 
+void CPTValue::write(IFileIO *out, offset_t offset)
+{
+    size32_t dataSz = (size32_t)length();
+    // NB: same format as CPTValue::sserialie
+    out->write(offset, sizeof(dataSz), &dataSz); offset += sizeof(dataSz);
```

Comment:
I think it was left as LZW to avoid backward compatibility issues,
but since this change in introduces a backward compatibility threshold itself, it may make sense to swap the compression method for new values (and use a different extension, to differentiate legacy vs new).
I'll look into it.